docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -141,6 +141,8 @@ The following provides a good starting point for creating ``etc/jvm.config``:
     -XX:PerBytecodeRecompilationCutoff=10000
     -Djdk.attach.allowAttachSelf=true
     -Djdk.nio.maxCachedBufferSize=2000000
+    -XX:+UnlockDiagnosticVMOptions
+    -XX:+UseAESCTRIntrinsics
```

Comment:
Here we can have a code comment too, for the benefit of our users.

Reply:
ok